### DISCUSSION TITLE
Heroku does it work?

### DISCUSSION THREAD

**Discussion by tomicOgnjen** [drmohundro/SWXMLHash#264] [General]
Hi guys, 
wanted to know is there any issues using SWXMLHash on Heroku with Vapor?

**Comment by drmohundro** [comment]
I've honestly never tried it... the only dependencies it has are either `Foundation` or `Foundation` plus `FoundationXML` when on Linux. I'd guess it should work fine.

**Comment by tomicOgnjen** [comment]
Hi, tnx for reply 🍻

I forgot to update this, it is working 

Note:
in import section of the file, you need to specify following:

#if canImport(FoundationNetworking)
import FoundationNetworking
#endif

After that, all good, runs smoothly

**Reply by drmohundro** [reply]
Awesome, thanks for following up - I'll add that note to the README.
